doc/user/content/sql/alter-cluster.md

```diff
@@ -6,44 +6,159 @@ menu:
 
-#### Set new configuration value
+`ALTER CLUSTER` has the following syntax forms:
 
-{{< diagram "alter-cluster-set.svg" >}}
+{{< tabs >}}
+{{< tab "Set a configuration" >}}
 
-#### Reset configuration to original value
+To set a cluster configuration:
 
-{{< diagram "alter-cluster-reset.svg" >}}
+  ```mzsql
+  ALTER CLUSTER <cluster_name>
+  SET (
+      SIZE = <text>,
+      REPLICATION FACTOR = <int>,
+      INTROSPECTION INTERVAL = <interval>,
+      INTROSPECTION DEBUGGING = <bool>,
+      MANAGED = <bool>,
+      SCHEDULE = { MANUAL | ON REFRESH (...) }
+  )
+  [WITH ({ WAIT UNTIL READY({TIMEOUT | ON TIMEOUT {COMMIT|ROLLBACK}}) | WAIT FOR <duration> })]
+  ;
+  ```
 
-### Cluster options
+You must have ownership of the cluster. See also [Required
+privileges](#required-privileges).
 
-{{% cluster-options %}}
+{{< /tab >}}
+{{< tab "Reset to default" >}}
 
-### `with_options`
+To reset a cluster configuration back to its default value:
 
-{{< diagram "alter-cluster-set-with-options.svg" >}}
+  ```mzsql
+  ALTER CLUSTER <cluster_name>
+  RESET (
+      REPLICATION FACTOR,
+      INTROSPECTION INTERVAL,
+      INTROSPECTION DEBUGGING,
+      MANAGED,
+      SCHEDULE
+  )
+  ;
+  ```
 
-Field                         | Value                 | Description
-------------------------------|-----------------------|-------------------------------------
-**WAIT UNTIL READY...TIMEOUT**                   | `duration`            | For a [graceful cluster resizing](#graceful-cluster-resizing), the maximum duration to wait for the new replicas to be ready.
-**WAIT UNTIL READY...ON TIMEOUT**                | [`COMMIT`,`ROLLBACK`] | For a [graceful cluster resizing](#graceful-cluster-resizing), the action to take on timeout. <br><ul><li>`COMMIT` cuts over to the new replica regardless of its hydration status, which may lead to downtime.</li><li>`ROLLBACK` removes the pending replica and returns a timeout error.</li></ul>Default: `COMMIT`.
-**WAIT FOR**                  | `duration`            | For a [graceful cluster resizing](#graceful-cluster-resizing), a fixed duration to wait for the new replicas to be ready. This option might lead to downtime, so we recommend using the `WAIT UNTIL READY` option instead.
+You must have ownership of the cluster. See also [Required
+privileges](#required-privileges).
 
-## Graceful cluster resizing
+{{< /tab >}}
+{{< tab "Rename to" >}}
+
+To rename a cluster:
+
+  ```mzsql
+  ALTER CLUSTER <cluster_name> RENAME TO <new_cluster_name>;
+  ```
+
+You must have ownership of the cluster. See also [Required
+privileges](#required-privileges).
+
+{{< /tab >}}
+{{< tab "Alter owner to" >}}
+
+To change the owner of a cluster:
+
+  ```mzsql
+  ALTER CLUSTER <cluster_name> OWNER TO <new_owner_role>;
+  ```
+
+To rename a cluster, you must have ownership of the cluster and membership in
+the `<new_owner_role>`. See also [Required privileges](#required-privileges).
+
+{{< /tab >}}
+{{< tab "Swap names with another cluster" >}}
```

Comment:
This isn't an operation that a user would run manually. This should be contextualized within blue/green deployments.

Reply:
I think we discussed this elsewhere in the PR, but I'd +1 having the full syntax listed under `Syntax`, and instead provide examples for the operations each syntax variation allows you to perform under `Examples`.

Reply:
- Swap names: I did wonder why we had a whole page for this.  Will blurb that it's for blue/green and included just for completeness.

- As for the listing the full syntax, as playing around with this particular command, I think for a small subset of our commands (like Alter cluster), it may be easier for our users to have them separated for now since less to tweak after copying.  As I work through other pages, it may turn out that I find a way that won't be cumbersome for our users to have everything in one and will retrofit.
  - I'm strongly **+1** for examples for each variation -- but to keep balls rolling on other work, didn't do it in this PR.

Reply:
Oh ... and later on,  by the time I get to the `ALTER SCHEMA` page, it might be that we may want to remove the `swap with` part wholesale from our user facing docs.